docs/content/usages/promise.md

```diff
@@ -0,0 +1,49 @@
+---
+sort: 2
+---
+
+# Action Promise
+
+Nikita's actions always return [JavaScript Promise](https://nodejs.dev/learn/understanding-javascript-promises) and provide the guarantee that all actions are executed sequentially according to their declaration.
+
+## Accessing the action output
+
+The [action output](/current/action/output) is returned after its Promise is fulfilled.
+
+To access the result, you can use the [`then` method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/then) of Promise and pass a callback function. You also can call the [`finally` method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/finally) and pass a callback function with JavaScript commands to run them synchronously after the Nikita actions:
+
+```js
+nikita
+.call(function() {
+  // Call an action
+  this.call(() => 'done')
+  // Access the action output
+  .then((result) => {
+    console.info(result)  // done
+  })
+})
+// Run synchronously after Nikita actions
+.finally(() => {
+  console.info('finally done')
+})
+```
+
+There is also another approach to access the action output using the [`async`/`await` operators](https://nodejs.dev/learn/modern-asynchronous-javascript-with-async-and-await).
```

Comment:
Since this is the most elegant and relevant approach, start with it. The, after it, you can start the next section with "Alternatively, this exemple can be rewritten using the Promise API.."